test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -4621,6 +4624,7 @@ public async Task DurableEntity_CleanEntityStorage_Many(string storageProvider)
                 var client = await host.StartOrchestratorAsync(nameof(TestOrchestrations.CreateEmptyEntities), entityIds, this.output);
                 var status = await client.WaitForCompletionAsync(this.output);
 
+                // TODO: try removing

```

Comment:
could you please clarify/ expand on this comment?

Reply:
Woops, I meant to try removing that as a part of this PR. I'm just trying to remove delays unless they are really necessary.